connector_ecommerce/product.py

```diff
@@ -137,25 +137,25 @@ def create(self, vals):
         return product
 
 
-class ProductPriceType(models.Model):
-    _inherit = 'product.price.type'
-
-    pricelist_item_ids = fields.One2many(
-        comodel_name='product.pricelist.item',
-        inverse_name='base',
-        string='Pricelist Items',
-        readonly=True,
-    )
-
-    @api.model
-    def sale_price_fields(self):
-        """ Returns a list of fields used by sale pricelists.
-        Used to know if the sale price could have changed
-        when one of these fields has changed.
-        """
-        item_model = self.env['product.pricelist.item']
-        items = item_model.search(
-            [('price_version_id.pricelist_id.type', '=', 'sale')],
-        )
-        types = self.search([('pricelist_item_ids', 'in', items.ids)])
-        return [t.field for t in types]
+# class ProductPriceType(models.Model):
```

Comment:
I agree with @pedrobaeza Why do you remove this code. It seems that you remove part of the module functionalities.